packages/launchpad-navigator/package.json

```diff
@@ -33,6 +33,7 @@
 		"@automattic/data-stores": "workspace:^",
 		"@automattic/i18n-utils": "workspace:^",
 		"@automattic/launchpad": "workspace:^",
+		"@wordpress/components": "^25.9.1",
```

Comment:
This looks like it's for `packages/launchpad-navigator`, not `packages/launchpad`.